spe_utils/utils.py

```diff
@@ -6,6 +6,10 @@
 API_DIR_SECRET = 'api_dir_secret'
 API_DIR_URL = 'api_dir_url'
 COURSE_ID = 'course'
+ENV = 'env'
```

Comment:
This file should probably be renamed to constants.py as it seems to only contain constants

Reply:
Done